src/vivarium/examples/boids/neighbors.py

```diff
@@ -15,17 +15,18 @@ class Neighbors(Component):
     ##############
     @property
     def configuration_defaults(self) -> Dict[str, Any]:
-        return {"neighbors": {"radius": 10}}
+        return {"neighbors": {"radius": 60}}
 
     @property
     def columns_required(self) -> Optional[List[str]]:
-        return ["x", "y"]
+        return ["x", "y", "color"]
```

Comment:
Where is `color` used?

Reply:
Nowhere 😅 This was left over from a previous version; I'll remove it.